proposals/handler.go

```diff
@@ -180,27 +179,53 @@ func (h *Handler) HandleSyncedBallot(ctx context.Context, expHash types.Hash32,
 	return nil
 }
 
-// collectHashes gathers all hashes in a proposal or ballot.
-func collectHashes(a any) []types.Hash32 {
-	p, ok := a.(types.Proposal)
-	if ok {
-		hashes := collectHashes(p.Ballot)
-		return append(hashes, types.TransactionIDsToHashes(p.TxIDs)...)
+func (h *Handler) HandleActiveSet(ctx context.Context, id types.Hash32, peer p2p.Peer, data []byte) error {
```

Comment:
should disallow any ref ballot without empty active set before the scheduled layer.

Reply:
i don't think that such code is doing anything useful

Reply:
i added that condition